### PR TITLE
copied core code from CS repo

### LINKED ISSUE #3: Transfer old code to new repo
move from 21430-Centerstage git repo into this one

### PR DESCRIPTION
**Pull request by FriendlyGuardian**
And so it begins!!!

Before issuing a pull request, please see the contributing page.

fixes #3

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    private boolean CurrentAlign = true;
+    private boolean DriverOrientationDriveMode = true;
+
+    public double derivativeConstantAngle;
+    public double proportionalConstantAngle;
+
+    public Telemetry telemetry = null;
+
+    private boolean resettingImu = false;
+    private double AutoStartAngle = 0;
```
Comment (JenX1) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java:43]: Are all of these needed?

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java
Comment (JenX1) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java:1]: This file needs more comments

**Review by JenX1 — APPROVED** [review]
Good code, 👍

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java (hunk 1 above)
Comment (FriendlyGuardian) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java:43]: To my understanding yes, there maybe a few that could move down into the ITDbot class.

Comment (JenX1) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java:43]: Sounds great!

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java
Comment (FriendlyGuardian) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Robot.java:1]: Yeah that's true 😞

**Pull request merged**